### PR TITLE
Move Fahrenheit Variable Below Comment Line

### LINKED ISSUE #15664: Move Fahrenheit Variable Below Comment Line
#### Convert Celsius to Fahrenheit
https://www.freecodecamp.org/challenges/convert-celsius-to-fahrenheit

#### Issue Description
There's an inconsistency between the forum solution to this module and the needed solution. In order to get the tests to pass, you need to modify the Fahrenheit variable, but it's above the comment that says to only modify code below that comment. 

My proposed solution is to update the content on the exercise so the variable is below the comment, so it would appear like this: 

```
function convertToF(celsius) {
  // Only change code below this line
  
  var fahrenheit 
  
  // Only change code above this line
  return fahrenheit;
}

// Change the inputs below to test your code
convertToF(30);
function convertToF(celsius) {
  var fahrenheit = (celsius * 9/5) + 32;
  // Only change code below this line
  
  // Only change code above this line
  return fahrenheit;
}

// Change the inputs below to test your code
convertToF(30);
```

Here's a link to the solution in the forum so you can see the discrepency: https://forum.freecodecamp.org/t/freecodecamp-challenge-guide-convert-celsius-to-fahrenheit/16806

#### Browser Information

* Browser Name, Version: Chrome
* Operating System: Mac OS Sierra
* Mobile, Desktop, or Tablet: Desktop

#### Your Code
This is what I used to get the tests to pass. I had to modify ```var fahrenheit``` which is above the comment line. My sense is this is where a lot of the confusion from this exercise is coming from. 

```
function convertToF(celsius) {
  var fahrenheit = (celsius * 9/5) + 32;
  // Only change code below this line
  
  // Only change code above this line
  return fahrenheit;
}

// Change the inputs below to test your code
convertToF(30);
```
#### Screenshot

Notice how the comments between the exercise and the solution are not in the same place. This is what's creating the confusion because in order to solve the problem you need to modify the ```var``` that's above the comment in the exercise.  

screen shot 2017-07-18 at 10 55 35 pm

screen shot 2017-07-18 at 10 56 14 pm

### PR DESCRIPTION
**Pull request by prateekgoel**
Challenge URL: https://www.freecodecamp.org/challenges/convert-celsius-to-fahrenheit

#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.
- [x] Closes currently open issue : Closes #15664 

#### Description

Changed the desciption as mentioned in the linked issue.

### CODE REVIEW

**Comment by dhcodes** [comment]
@prateekgoel could you also add Eric's note from the issue thread? You can go back to your branch, add the note, save the file, and then run `git add .`, `git commit --amend`, save the file as-is, and then run `git push origin fix/move-fahrenheit-variable -f` to update this PR.

File: seed/challenges/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting.json [hunk 1]
```diff
@@ -104,7 +104,7 @@
       "title": "Convert Celsius to Fahrenheit",
       "description": [
         "The algorithm to convert from Celsius to Fahrenheit is the temperature in Celsius times <code>9/5</code>, plus <code>32</code>.",
-        "You are given a variable <code>celsius</code> representing a temperature in Celsius. Use the variable <code>fahrenheit</code> already defined and apply the algorithm to assign it the corresponding temperature in Fahrenheit."
+        "You are given a variable <code>celsius</code> representing a temperature in Celsius. Use the variable <code>fahrenheit</code> already defined and assign it the Fahrenheit temperature equivalent to the given Celsius temperature. Use the algorithm mentioned above to help convert the Celsius temperature to Fahrenheit."
```
Comment (dhcodes) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting.json]: Add @erictleung's note after these lines.

**Review by dhcodes — CHANGES_REQUESTED** [review]

File: seed/challenges/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting.json (hunk 1 above)
Comment (prateekgoel) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting.json]: Ok. I will do that.

Comment (prateekgoel) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-algorithm-scripting.json]: Ok. I'll add that.

**Comment by prateekgoel** [comment]
@dhcodes Add the note.

**Review by dhcodes — APPROVED** [review]

**Comment by dhcodes** [comment]
@prateekgoel I forgot to mention this, but in the future, please squash your commits. Read about it here: https://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231

Alternatively, you could amend your past changes and then force push as I explained in the comment above. This would've kept these changes as one commit.

**Comment by prateekgoel** [comment]
@dhcodes Sure. I'll take care in future.

**Pull request merged**